docs/scripts/utils.py

```diff
@@ -32,9 +32,15 @@ def adjust_pictures(
 
```

Comment:
question: the mechanism above in lines 27-31, do I understand it correctly that this overall function only finds the very first match of the pattern and ignores potential subsequent ones?

Reply:
Currently yes.

Reply:
hmm good question :D 
conceptually it should replace all occurrences I think, but in practice that would mean we have a doc that has the same picture twice... doesnt seem likely to me
hence I will resolve